diff --git a/typings/index.d.mts b/typings/index.d.mts
index ad0b67870da010ffeb333ef06a67465b26e597b9..a1dee9410baf284b00c5c9b46e81e59478fafb36 100644
--- a/typings/index.d.mts
+++ b/typings/index.d.mts
@@ -12612,7 +12612,6 @@ export class User extends Base {
    * The tag of this user
    * <info>This user's username, or their legacy tag (e.g. `hydrabolt#0001`)
    * if they're using the legacy username system</info>
-   * @deprecated Use {@link User.username} instead.
    */
   public get tag(): string;
 
diff --git a/typings/index.d.ts b/typings/index.d.ts
index e0e47f7eb14045140ea8ea2dd1c4e40482668294..672de5f8b8b7f3a6babf7f96b94fbdab710c5ac0 100644
--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -12612,7 +12612,6 @@ export class User extends Base {
    * The tag of this user
    * <info>This user's username, or their legacy tag (e.g. `hydrabolt#0001`)
    * if they're using the legacy username system</info>
-   * @deprecated Use {@link User.username} instead.
    */
   public get tag(): string;
 
