diff --git a/types/index.d.ts b/types/index.d.ts
index 98ebe711148cbfd499da864118e0cabd1e2c9601..f17239d2d0bf9db11e3b4efa764a7e1eaef3daa9 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -2,7 +2,7 @@ import { Decimal } from 'decimal.js'
 
 declare const math: math.MathJsStatic
 export as namespace math
-export = math
+export default math
 
 type NoLiteralType<T> = T extends number
   ? number
